src/modules/data/sound.c

```diff
@@ -25,13 +25,15 @@ struct Sound {
   uint32_t sampleRate;
   uint32_t frames;
   uint32_t cursor;
+  void *callbackMemo;
+  SoundDestroyCallback callbackMemoDestroy;
```

Comment:
Note: A future plan that I may implement is adding a typed union (SoundType enum plus a union for callbackMemo/Destroy and stream and decoder).  This wouldn't have any impact on the public API

Reply:
Makes sense. I could imagine a callback making use of decoder, but blob less likely.